.github/workflows/linux-qe.yml

```diff
@@ -0,0 +1,150 @@
+name: tester-linux
+
+on:
+  workflow_run:
+    workflows: [Build Windows artifacts]
+    types:
+      - completed
+
+jobs:
+  tests:
+    runs-on: [self-hosted, linux, testing-farm]
+    strategy:
+      fail-fast: false
+      matrix:
+        go: ['1.21']
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: ${{ matrix.go }}
+      
+      - name: prepare env
+        run: |
+          sudo yum install podman openssh-server git make -y
+          echo "commit_sha=${commit_sha}" >> "$GITHUB_ENV"
+          mkdir ${{ env.commit_sha }}
+          cd ${{ env.commit_sha }}
+
+          #timestamp=`date +%s`
+          #mkdir $timestamp
+          #echo "timestamp=$timestamp" >> $GITHUB_ENV
+          $cd $timestamp
+
+          ssh-keygen -t rsa -N '' -f id_rsa -q 
+          eval $(ssh-agent -s)
+          echo $SSH_AUTH_SOCK > ssh_auth_sock
+          echo $SSH_AGENT_PID > ssh_agent_pid
+          #kill $(cat ssh_agent_pid)
+          ssh-add id_rsa
+
+      - name: reserve machine from testing farm
+        working-directory: ${{ env.timestamp }}
+        env: 
+          TESTING_FARM_API_TOKEN: ${{ secrets.TESTING_FARM_API_TOKEN }}
+          PULL_SECRET: ${{ secrets.PULL_SECRET }}
+        run: |
+          echo "${PULL_SECRET}" > pull-secret
+          export TESTING_FARM_API_TOKEN=${TESTING_FARM_API_TOKEN}
+          #export TESTING_FARM_API_TOKEN=4a7ba1e1-3881-4bf8-9543-b3521b4fed17
```

Comment:
I guess this token has been reverted. The new one I guess it is already set on upstream crc repo right?